Formula/govendor.rb

```diff
@@ -0,0 +1,36 @@
+class Govendor < Formula
+  desc "Go vendor tool that works with the standard vendor file."
+  homepage "https://github.com/kardianos/govendor"
+  url "https://github.com/kardianos/govendor/archive/v1.0.3.tar.gz"
+  sha256 "7d7a032355ee4117822150db72a9db745d9a147a2e32972e58a65fe7545444f2"
+  head "https://github.com/kardianos/govendor.git"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+
+    (buildpath/"src/github.com/kardianos/").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/kardianos/govendor"
+    system "go", "build", "-o", bin/"govendor"
+  end
+
+  test do
+    # Default HOMEBREW_TEMP is /tmp, which is actually a symlink to /private/tmp.
+    # `govendor` bails without `.realpath` as it expects $GOPATH to be "real" path.
+    ENV["GOPATH"] = testpath.realpath
+    (testpath/"src/github.com/project/testing").mkpath
```

Comment:
Or if you'd like to keep it mostly as is, which I think is fine, I'd probably at least factor out `commit`:
```
test do
  ENV["GOPATH"] = testpath.realpath
  commit = "89d9e62992539701a49a19c52ebb33e84cbbe80f"
  (testpath/"src/github.com/project/testing").mkpath
  cd testpath/"src/github.com/project/testing" do
    system bin/"govendor", "init"
    system bin/"govendor", "fetch", "-tree", "golang.org/x/crypto@#{commit}"
    assert_match commit, File.read("vendor/vendor.json")
    assert_match "golang.org/x/crypto/blowfish", shell_output("#{bin}/govendor list")
  end
end
```